docs/iot-video-tutorials.md

```diff
@@ -57,3 +57,57 @@ You will learn how to:
         <iframe src="https://www.youtube.com/embed/JCW_hShAp7I" frameborder="0" allowfullscreen></iframe>
     </div>
 </div>
+
+
+#### Dashboard development guide, Part 1: Visualizing Assets data using Maps and Tables
+
+The first part of this tutorial provides preparations for the creating a new dashboard, visualizing attributes data.
```

Comment:
The first part of this tutorial provides preparations for the creating a new dashboard and visualizing data from asset attributes.